recipes/searchgui/meta.yaml

```diff
@@ -15,26 +15,15 @@ source:
   sha256: 39a37c23a39ad1ab4dd1868dac7421d6ac5e612c012517cc0b7a5a3192ff79bc
 
 requirements:
-  #run_exports:
-  #  strong:
-  #    - zlib
-  #    - libstdcxx-ng
-  #    - expat
+  
   build:
     - {{ compiler('c') }}
-    #- {{ compiler('cxx') }}
+  host:
```

Comment:
Thanks @dpryan79 

Ah, I see. Of course we can set the link without having xtandem installed. Its then just a stale link during building. But then it should also not be required in the host requirements, or? 

But tests still don't seem to run...